server/routes/about.js

```diff
@@ -1,20 +1,9 @@
 var express = require('express');
 var router = express.Router();
-var database = require('../config/database');
 
-router.get('/', function(req, res) {
-
-	var aboutData = function(err, results){
-		console.log("result: "  + results);
-		if(err) 
-			throw err;
-		else
-			res.render('about', {about: results});
-	};
-
-	//Pass a callback to getAbout function to fetch About data
-	database.getAbout(aboutData);
+// GET About page
+router.get('/', function(req, res){
+	res.render('about', {title: 'About'});
```

Comment:
Is there any reason to pass data to the app on render if you're using Flux/React on the client side for all rendering?